pkg/uplinkdb/uplinkdb.go

```diff
@@ -0,0 +1,43 @@
+// Copyright (C) 2018 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package uplinkdb
+
+import (
+	"context"
+	"crypto"
+	"time"
+
+	"go.uber.org/zap"
+)
+
+// DB stores bandwidth agreements.
+type DB interface {
+	// SavePublicKey adds a new bandwidth agreement.
+	SavePublicKey(context.Context, Agreement) error
+	// GetPublicKey gets the public key of uplink corresponding to serial number
```

Comment:
Comment needs to be updated, no serial numer in params.

Reply:
Done